### PR TITLE
Implementing Screen Reader Accessibility and Enhanced Error Message Handling with languages

### PR DESCRIPTION
**Pull request by andgar2010**
(This is a kinda feature request so feel free to reject if you don't like!)

improvements to accessibility for screen readers, enhancing the use of alternative text for images, and position of image. 
[Source](https://html.spec.whatwg.org/multipage/images.html#alt)

In addition, the language of error messages has been refined for greater clarity and precision.

Note:
I'm sorry, I don't speak Japanese. 
I have used an automatic translator for your language.
If there are any mistakes, please correct this file in your language.

### CODE REVIEW

**Comment by kawamataryo** [comment]
@andgar2010 
Sorry for the late reply 🙇‍♂️
This is great! Thank you for your pull request.
I will review it.

File: src/lib/rewriteReadmeToIncludeCardText.ts [hunk 1]
```diff
@@ -26,7 +48,9 @@ const createCardText = ({
 
   const updateTime = showUpdateTime ? `  \nLast Updated on ${new Date().toLocaleString()}` : ''
 
-  return `<a href="https://lapras.com/public/${shareId}" target="_blank" rel="noopener noreferrer"><img src="${imageUrl}" width="${cardWidth}" ></a>${updateTime}`
+  const _alternativeText: string = getAlternativeText({ alternativeText, lang, shareId, score })
+
+  return `<a href="https://lapras.com/public/${shareId}" target="_blank" rel="noopener noreferrer"><img ${isCenter ? 'align="center"' : ''} alt="${_alternativeText}" src="${imageUrl}" width="${cardWidth}" ></a>${updateTime}`
```
Comment (kawamataryo) [inline src/lib/rewriteReadmeToIncludeCardText.ts:53]: Thanks for adding a great option!

However, Even if I put align="center" on the img element, it is not centered. If you want to center it, you must align the parent block element.

```suggestion
  return `<p ${isCenter ? 'align="center"' : ''}><a href="https://lapras.com/public/${shareId}" target="_blank" rel="noopener noreferrer"><img alt="${_alternativeText}" src="${imageUrl}" width="${cardWidth}" ></a>${updateTime}</p>`
```

File: README.md [hunk 2]
```diff
@@ -74,6 +77,8 @@ jobs:
 |LANG|`ja`|スコアの表記方法（`ja` or `en`）|
 |CARD_WIDTH|`400`|出力する画像のwidth|
 |UPDATE_TIME|`true`|スコアの更新日時の表示有無|
+|IS_CENTER|`true`|スコアの更新日時の表示有無|
```
Comment (kawamataryo) [inline README.md:80]: Corrected to natural Japanese.

```suggestion
|IS_CENTER|`true`|カード画像を中央揃えするかどうか|
```

File: README.md [hunk 3]
```diff
@@ -74,6 +77,8 @@ jobs:
...
+|ALTERNATIVE_TEXT|`${shareId}のLAPRASでのスコアは次の通りです: エンジニアリング: ${score.eScore} / 5.0, ビジネス: ${score.bScore} / 5.0, インフルエンス: ${score.iScore} / 5.0。`|画像の代替テキスト、スクリーンリーダーの利便性向上に必須。|
```
Comment (kawamataryo) [inline README.md:81]: Corrected to natural Japanese.

```suggestion
|ALTERNATIVE_TEXT|`${shareId}のLAPRASでのスコアは次の通りです: エンジニアリング: ${score.eScore} / 5.0, ビジネス: ${score.bScore} / 5.0, インフルエンス: ${score.iScore} / 5.0。`|カード画像の代替テキスト、スクリーンリーダーでの利便性向上に関係|
```

**Comment by andgar2010** [comment]
Dear @kawamataryo,

I want to thank you for your time and patience in reviewing my Pull Request. Although the PR is not extensive, I understand that reviewing can take time, so I highly appreciate the attention you've given it.

Additionally, I am thankful for the corrections you have provided on some lines of code. If there is anything additional that needs adjusting, please let me know.

Lastly, I hope you find my Pull Request suitable and that it can be approved. Your feedback is invaluable and I am willing to make the necessary corrections to ensure it meets the project’s standards.

Once again, thank you for your response and review.

Warm regards,

**Comment by andgar2010** [comment]
Already Releases updated?

**Comment by kawamataryo** [comment]
@andgar2010 

I've just released v1.3.0. Thank you for your contributions! 🎉

**Pull request merged**